### ISSUE TITLE
ListView: Clicking on the selection checkbox gets ignored every other click

### ISSUE THREAD

**Issue by jseanxu** [winjs/winjs#1195]
## Repro
* Create a ListView in ListLayout with `selectionMode = "multi"` and `tapBehavior = "toggleSelect"`
* Click directly on the selection checkbox in the corner of a ListView item
* Click a few more times and observe

#### Expected Behavior
The selection toggles with every click.

#### Actual Behavior
The selection toggles every other click.

**Comment by AmazingJaze** [comment]
@jseanxu  is this a dupe of #1193?

**Comment by jseanxu** [comment]
Yep, closing this one.

**Comment by TheWrathOfConst** [comment]
The problem is coming from `ItemEventsHandler_onPointerUp`. The first thing this function does when handling pointerup is get the element that was released:

```js
                    var touchInput = (eventObject.pointerType === PT_TOUCH),
                        releasedElement = this._releasedElement(eventObject),
                        releasedIndex = site.indexForItemElement(releasedElement),
                        releasedHeaderIndex = releasedElement && _ElementUtilities.hasClass(releasedElement, _Constants._headerContainerClass) ? site.indexForHeaderElement(site.pressedHeader) : site.indexForHeaderElement(releasedElement);
```

The call to `this._releasedElement(eventObject)` is our problem. All it does is get the element beneath the cursor:
```js
                _releasedElement: function ItemEventsHandler_releasedElement(eventObject) {
                    return _Global.document.elementFromPoint(eventObject.clientX, eventObject.clientY);
                },
```
This is a problem because elementFromPoint is transform aware, and we're applying some substantial rotation transforms to ListView items when they're clicked on their edges. If you click an item too close to an edge, then the rotation can be enough to rotate the item such that it's no longer under the mouse cursor. Then, when the cursor is released, _releasedElement tries to get the element directly below it, finds empty space, and decides no element was affected and the ListView doesn't try to select anything.

Turns out this is an old bug that's been in WinJS since 1.0. The original bug was resolved as pri 3/won't fix, since at the time it wasn't so bad (you had to click 1px into the of an edge of an item to hit the bug). Now that we're applying transforms that are much larger than the scale we used to use, this bug is important to fix.

Going through the source history, it looks like the reason we added this releasedElement stuff was to support cancelation of a click. We had a bug where you could click on an invokable item, then move the cursor away from that item, then release, and the item would fire an itemInvoked event, when instead nothing should have happened. This releasedElement check was added to make sure the cursor was still over the item that was originally clicked before firing an invoke event. When fixing this, we need to make sure not to regress the original bug that was fixed by the releasedElement code

**Comment by jseanxu** [comment]
Temp fixed via 50619e1be1d06f7d5fdc30309f2445243649f4de. See #1215 for tracking a permanent fix.
